projects/angular/src/forms/combobox/options.ts

```diff
@@ -43,8 +43,8 @@ let nbOptionsComponents = 0;
     </div>
 
     <!-- Rendered if data set is empty -->
-    <div *ngIf="emptyOptions">
-      <span class="clr-combobox-options-empty-text">
+    <div *ngIf="emptyOptions" aria-label="">
+      <span class="clr-combobox-options-empty-text" [attr.aria-label]="commonStrings.keys.comboboxNoResults">
```

Comment:
```suggestion
    <div *ngIf="emptyOptions" [attr.aria-label]="commonStrings.keys.comboboxNoResults">
      <span class="clr-combobox-options-empty-text">
```

Does this not work? I think it's odd to have an empty `aria-label`.